stepper.md

```diff
@@ -42,23 +43,198 @@ motor drivers). Additional delay values can be added in the future.*
 9  [only when interface = 0x04] motorPin3 (0-127)
 10 [only when interface = 0x04] motorPin4 (0-127)
 11 END_SYSEX                         (0xF7)
-```
+-----------------------------------------------------
 
-Stepper step
 ```
-0  START_SYSEX          (0xF0)
-1  Stepper Command      (0x72)
-2  config command       (0x01)
-3  device number        (0-5)
-4  direction            (0-1) (0x00 = CW, 0x01 = CCW)
+
+###Stepper step
+```c
+ -----------------------------------------------------
+0  START_SYSEX          	(0xF0)
+1  Stepper Command      	(0x72)
+2  config command       	(0x01)
+3  device number        	(0-5)
+4  direction            	(0-1) (0x00 = CW, 0x01 = CCW)
 5  num steps byte1 LSB
 6  num steps byte2
 7  num steps byte3 MSB  (21 bits (2,097,151 steps max))
-8  speed LSB            (steps in 0.01*rad/sec  (2050 = 20.50 rad/sec))
-9  speed MSB
+8  [optional] speed LSB (steps in 0.01*rad/sec  (2050 = 20.50 rad/sec))
+9  [optional] speed MSB
 10 [optional] accel LSB (acceleration in 0.01*rad/sec^2 (1000 = 10.0 rad/sec^2))
 11 [optional] accel MSB
 12 [optional] decel LSB (deceleration in 0.01*rad/sec^2)
 13 [optional] decel MSB
-14 END_SYSEX            (0xF7)
+14 END_SYSEX            	(0xF7)
+-----------------------------------------------------
+
+```
+
+Proposals to ammend:
+---
+
+###Report Steppers's position
+
+This reports the steppers movement from the home point over time. At first the home position is the position the stepper was in when it was inited but can be set with the setHome proposal below 
+
+Query
+```c
+ -----------------------------------------------------
+0 START_SYSEX               (0xF0)
+1 Stepper Command           (0x72)
+2 Stepper Get Position      (0x02)
+3 Stepper #                 ([0 - MAX_STEPPERS-1])
+4 END_SYSEX                 (0xF7)
+-----------------------------------------------------
+
+```
+ 
+Response 
+```c
+ -----------------------------------------------------
+0 START_SYSEX               (0xF0)
+1 Stepper Command      		(0x72)
+2 Stepper Get Position      (0x02)
+3 Stepper #                 ([0 - MAX_STEPPERS-1])
+4 current position, bits 0-6
+5 current position, bits 7-13
+6 current position, bits 14-20
+7 current position, bits 21-27
+8 the signed value			(negative 0x00 or positive 0x01)
+9 END_SYSEX                 (0xF7)
+-----------------------------------------------------
+
+```
+
+###Report Steppers's Distance from target
```

Comment:
Yeah any actual distance would depend on the steps per revolution and the gearing involved so it would be difficult to retrieve anything without passing in parameters